### ISSUE TITLE
Missing import statement in the src/index.js file.

### ISSUE THREAD

**Issue by Kuljeet-123** [CleverTap/clevertap-react-native#431]
I encountered an issue where our app kept crashing after installing the package. Upon examining the code, I realized that there was a missing import in the library file `src/index.js`.

The current code is:
```
import { DeviceEventEmitter, NativeEventEmitter, NativeModules } from 'react-native';
const CleverTapReact = require('./NativeCleverTapModule').default;
const EventEmitter = Platform.select({
    ios: new NativeEventEmitter(CleverTapReact),
    android: DeviceEventEmitter
});
```

Ideally, it should be:
```
import { DeviceEventEmitter, NativeEventEmitter, NativeModules, Platform } from 'react-native';
const CleverTapReact = require('./NativeCleverTapModule').default;
const EventEmitter = Platform.select({
    ios: new NativeEventEmitter(CleverTapReact),
    android: DeviceEventEmitter
});
```

This resolves the crash issue.

**Comment by Anush-Shand** [comment]
@Kuljeet-123 
Thanks for raising this issue. We will be fixing this shortly in the upcoming release. Can you please confirm if your react native app is an expo-app?

**Comment by Anush-Shand** [comment]
Thanks for confirming @Kuljeet-123 
`clevertap-react-native` doesn't officially support `expo` yet and this issue is only faced on expo-apps

We will be fixing it the upcoming release nonetheless. The release is schedule before the end of this month
Thanks

**Comment by piyush-kukadiya** [comment]
Fixed in release [3.1.0](https://github.com/CleverTap/clevertap-react-native/releases/tag/3.1.0)
